docker/PythonDockerfile

```diff
@@ -37,6 +37,8 @@ RUN python3 -m spacy download de_core_news_sm \
     && python3 -m spacy download fr_core_news_sm \
     && python3 -m spacy download it_core_news_sm \
     && python3 -m spacy download sv_core_news_sm \
+    && python3 -m spacy download uk_core_news_sm \
+    && python3 -m spacy download en_core_web_sm \
+    && python3 -m spacy download el_core_news_sm \
     && python3 -m spacy download xx_ent_wiki_sm
 
-COPY ./tools /app
```

Comment:
I think this one was an accident.

Reply:
Yep, that merge had some side-effects. Fixed.